gio-sys/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir @ ea993ed)
+// This file was generated by gir (https://github.com/gtk-rs/gir @ a418513)
```

Comment:
gir sub-module updated to 6114d1c7e2, but here commit before merge.
I usually touch gir's `Cargo.toml` to regen after my gir commit merged to get right version.